### PR TITLE
LPS-147458

### PR DESCRIPTION
**Pull request by eliasantosf**
Bug Task: [LPS-147458](https://issues.liferay.com/browse/LPS-147458)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by cgmonte** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/SidebarHeader.js [hunk 1]
```diff
@@ -96,14 +96,24 @@ export default function SidebarHeader({
...
+				{showDeleteButton && title !== 'Assignments' && (
 					<ClayButtonWithIcon
 						className="text-secondary trash-button"
 						displayType="unstyled"
 						onClick={() => setShowDeleteConfirmationModal(true)}
 						symbol="trash"
 					/>
 				)}
+
+				{showDeleteButton && title === 'Assignments' && (
```
[the remark this answers is not in the document]
Comment (eliasantosf) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/SidebarHeader.js]: Done!

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/SidebarHeader.js [hunk 2]
```diff
@@ -96,14 +96,24 @@ export default function SidebarHeader({
...
+					<ClayButtonWithIcon
+						className="text-secondary trash-button"
+						disabled
```
[the remark this answers is not in the document]
Comment (eliasantosf) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/SidebarHeader.js]: Done!

**Comment by cgmonte** [comment]
Just started reviewing :)

**Review by cgmonte — APPROVED** [review]

**Comment by cgmonte** [comment]
ci:test:relevant

**Comment by cgmonte** [comment]
ci:forward

**Comment by cgmonte** [comment]
ci:stop

**Pull request closed without merging**